lib/seed_helper.rb

```diff
@@ -153,6 +153,16 @@ def report_templates
       Dir["#{Rails.root}/app/views/unattended/report_templates/*.erb"]
     end
 
+    def plugin_templates
+      default_dirs = Foreman::Plugin.all.flat_map { |plugin| Dir.glob(plugin.engine.root + 'app/templates') }
```

Comment:
You can simplify this to 
```suggestion
      default_dirs = Foreman::Plugin.all.map { |plugin| plugin.engine.root.join('app', templates') }
```

Then the glob at the end will take care of non-existing directories